tools/blue_deployment/helpers.js

```diff
@@ -0,0 +1,136 @@
+const colors = require('colors/safe');
```

Comment:
Instead of `helpers`, I would prefer meaningful file names. Those files should contain relevant classes or functions relevant to the name.